pkg/disk/utils.go

```diff
@@ -563,6 +563,12 @@ func getDiskVolumeOptions(req *csi.CreateVolumeRequest) (*diskVolumeArgs, error)
 	if err != nil {
 		return nil, fmt.Errorf("illegal required parameter type: %s", volOptions["type"])
 	}
+
+	if len(diskType) != 1 || diskType[0] != DiskRegional {
```

Comment:
It is better to check CategoryDesc.Regional, in case we have more regional disk category in the future.

```suggestion
	if slices.ContainsFunc(diskType, func(t Category) bool {return !AllCategories[t].Regional}) {
```